example/package.json

```diff
@@ -28,8 +29,7 @@
     "@types/jest": "link:../node_modules/@types/jest",
     "@types/node": "link:../node_modules/@types/node",
     "@types/react-dom": "link:../node_modules/@types/react-dom",
-    "@types/react-router-dom": "^5.3.2",
-    "@fortawesome/fontawesome-common-types": "^0.2.36"
```

Comment:
There was an error shown when I first ran `yarn install` inside example related to these common types.
Running install again doesn't show the error anymore and I couldn't reproduce it now.

Reply:
I'm not sure why that would be the case.. I think it might be that `yarn install` has to be ran at the top of the tree first.